aeron-archive/src/main/java/io/aeron/archive/ReplicationSession.java

```diff
@@ -767,7 +782,7 @@ private int replicate()
             }
 
             srcReplaySessionId = NULL_VALUE;
-            state(State.DONE);
+            state(State.DONE, "replicate");
```

Comment:
Use `isSync ? "sync" : "done"` instead of `"replicate"`. There is already a dedicated log event `logReplicationSessionDone` which will print full information.